includes/connect/class-wc-stripe-connect.php

```
@@ -263,37 +263,53 @@ private function save_stripe_keys( $result, $type = 'connect', $mode = 'live' )
-			$options[ $prefix . 'publishable_key' ]     = $publishable_key;
-			$options[ $prefix . 'secret_key' ]          = $secret_key;
-			$options[ $prefix . 'connection_type' ]     = $type;
-			$options['pmc_enabled']                     = 'connect' === $type ? 'yes' : 'no'; // When not connected via Connect OAuth, the PMC is disabled.
-			$should_default_optimized_checkout_on       = get_option( 'wc_stripe_optimized_checkout_default_on' );
+			$publishable_key = $result->publishableKey; // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
+			$secret_key      = $result->secretKey; // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
+			$is_test         = 'live' !== $mode;
+			$prefix          = $is_test ? 'test_' : '';
+			$default_options = $this->get_default_stripe_config();
+			// Ensure defaults are applied to the settings singleton.
+			$settings = WC_Stripe_Settings::get_instance();
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Settings' in method 'save_stripe_keys'. (undefined)

(StaticAccess)